host/xtest/regression_1000.c

```diff
@@ -1377,6 +1377,7 @@ static void xtest_tee_test_1016(ADBG_Case_t *c)
 	TEEC_Session session = { };
 	TEEC_Operation op = TEEC_OPERATION_INITIALIZER;
 	uint32_t ret_orig = 0;
+	int dummy;
```

Comment:
Right. As per Linux coding standard, not initializing it would have been fine. But I forgot that OP-TEE introduced an extra rule to force such default initialization: https://optee.readthedocs.io/en/latest/general/coding_standards.html#coding-standards.

I will pay more attention on this point.